packages/next/pages/_document.tsx

```diff
@@ -212,6 +213,54 @@ export class Head extends Component<
     return cssLinkElements.length === 0 ? null : cssLinkElements
   }
 
+  getInlineCSS(): JSX.Element[] | null {
+    const { assetPrefix, distDir, files } = this.context._documentProps
+    const { _devOnlyInvalidateCacheQueryString } = this.context
+    const cssFiles =
+      files && files.length ? files.filter((f) => /\.css$/.test(f)) : []
+
+    const inlineCssElements: JSX.Element[] = []
+    cssFiles.forEach((file) => {
+      try {
+        const contents = readFileSync(
```

Comment:
This does not work with the serverless target unless the static files are bundled